docs/_source/_common/snippets/start_page.md

```diff
@@ -1,32 +1,31 @@
-# Getting started with Argilla
+Welcome to Argilla! 👋😁 We're thrilled to have you on board. Beginning with Argilla is as simple as it gets, and you've successfully taken the first step!
 
-Welcome to Argilla! 👋😁 We're thrilled to have you on board. Starting with Argilla is as simple as it gets, and you've successfully taken the first step!
+To start using Argilla, you need to create your first dataset. Let's get started. 🚀
 
-To start annotating your data, you need to create a `FeedbackDataset`. Let's set things in motion. 🚀
+[For in-depth user guides, check the documentation 📖.](https://docs.argilla.io/en/latest/)
 
-[For a more in-depth understanding of Argilla, you can check our documentation 📖.](https://docs.argilla.io/en/latest/)
+# 1. Open an IDE, Jupyter or Collab
```

Comment:
I would try to use h2 (##) instead of h1 (let's see what Leire says but I'll start with that=